mariadb.html.md.erb

```diff
@@ -57,13 +57,36 @@ Cloud Foundry: [Spring Music Example](https://github.com/cloudfoundry-samples/sp
 
 ## <a id='limitations'></a> Known Limitations
 
-Our HA MariaDB offering is based on Galera, which brings some [known limitations](https://mariadb.com/kb/en/mariadb/mariadb-galera-cluster-known-limitations/) with it. Please read this article carefully.
+Our HA MariaDB offering is based on Galera, which brings some [known limitations (from mariadb.com)](https://mariadb.com/kb/en/mariadb/mariadb-galera-cluster-known-limitations/) with it. Please read this article carefully.
 
-Our Galera aaS is a shared cluster. Each customer has their own database. Put simply, the service broker executes the following statements:
+A summary of Galera limitations is on [Galera Caveats](http://mysql.rjweb.org/doc.php/galera) (blockquote modified for our offering):
+
+> Dear Schema Designer:
+
+> - InnoDB only, always have PK.
+
+> Dear Developer:
+
+> - Check for errors, even after `COMMIT`.
+> - Moderate sized transactions.
+> - Don't make assumptions about `AUTO_INCREMENT` values other than uniqueness. Not monotonicity, not gapless. (This applies for non-Galera, but probably happens more often here.)
+- Handling of "critical reads" is quite different (arguably better).
+
+> Dear DBA:
+
+> - `ALTER`s are handled differently.
+> - `TRIGGER`s and `EVENT`s may need checking.
```

Comment:
what checking?

Reply:
I don't know myself. I will investigate and update text